pikitlib/pikitlib/ultrasonic.py

```diff
@@ -0,0 +1,76 @@
+import RPi.GPIO as GPIO
+import time
+import statistics
+
+#Class requirements (from WPILIB):
+# 
+
+
+class Ultrasonic:
+    def __init__(self):
```

Comment:
Take cues from the wpilib Ultrasonic and accept the trigger and echo pins as arguements.   You can have them have magic default values, but there should be an option to specify other pins so that other sonar units could be used.